### ISSUE TITLE
Backup error with version 1.3.4

### ISSUE THREAD

**Issue by prefislt** [MaxMaeder/DriveBackupV2#14]
Backups doesn't work with latest version (works fine with version 1.3.3). 
Server is on latest Paper 1.16.2 build.

![image](https://user-images.githubusercontent.com/1362964/92103439-a5518480-ede8-11ea-8f79-7df12841d8b5.png)

```
>.... [10:19:06 WARN]: Exception in thread "Thread-249" java.lang.IllegalStateException: Asynchronous command dispatch!
>.... [10:19:06 WARN]: at org.spigotmc.AsyncCatcher.catchOp(AsyncCatcher.java:15)
>.... [10:19:06 WARN]: at org.bukkit.craftbukkit.v1_16_R2.CraftServer.dispatchCommand(CraftServer.java:775)
>.... [10:19:06 WARN]: at ratismal.drivebackup.UploadThread.setAutoSave(UploadThread.java:620)
>.... [10:19:06 WARN]: at ratismal.drivebackup.UploadThread.run(UploadThread.java:126)
>.... [10:19:06 WARN]: at java.lang.Thread.run(Thread.java:748)

```

**Comment by coco0325** [comment]
same problem

**Comment by MaxMaeder** [comment]
Can I see one of your `config.yml`s?

**Comment by coco0325** [comment]
```
# 
# DriveBackupV2   -   by Max Maeder (MaxTheGinus)
delay: -1
backup-thread-priority: 0
keep-count: 5
local-keep-count: 0
backups-require-players: true
restart-after-backup: false
scheduled-backups: true
schedule-timezone: +08:00
backup-format-timezone: +08:00
backup-list:
- path: "lobby"
  format: "'lobby-'yyyy-M-d--HH-mm'.zip'"
  create: true
dir: backups
destination: backups/lobby
backup-schedule-list:
- days: 
  - "sunday"
  time: "05:00"
googledrive:
  enabled: true
onedrive:
  enabled: false
ftp:
  enabled: false
  hostname: ftp.example.com
  port: 21
  sftp: false
  ftps: false
  username: Username
messages:
  send-in-chat: true
  no-perm: You don't have permission to do this!
  backup-start: Creating backups, the server may lag for a little while...
  backup-complete: Backup complete,
  next-backup: the next backup is in %TIME minutes
  next-schedule-backup: the next backup is at %DATE
  next-schedule-backup-format: h:mm a EEE, MMM d O
  auto-backups-disabled: Automatic backups are disabled
advanced:
  metrics: true
  update-check: true
  suppress-errors: false
  message-prefix: '&6[&4DriveBackupV2&6] '
  default-message-color: '&3'
  date-language: en
  ftp-file-separator: /

```

**Comment by MaxMaeder** [comment]
Can you try this [bug fix](https://drive.google.com/file/d/1JIyETXC2RqibiYKaTv75UUz0kV4n9Eyf/view?usp=sharing).

**Comment by rrunswickjr** [comment]
Same issue, I will try the bug fix.

[08:41:40 WARN]: Exception in thread "Thread-746" java.lang.IllegalStateException: Asynchronous command dispatch!
[08:41:40 WARN]:        at org.spigotmc.AsyncCatcher.catchOp(AsyncCatcher.java:15)
[08:41:40 WARN]:        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.dispatchCommand(CraftServer.java:775)
[08:41:40 WARN]:        at ratismal.drivebackup.UploadThread.setAutoSave(UploadThread.java:620)
[08:41:40 WARN]:        at ratismal.drivebackup.UploadThread.run(UploadThread.java:126)
[08:41:40 WARN]:        at java.lang.Thread.run(Unknown Source)

Config:
# 
# DriveBackupV2   -   by Max Maeder (MaxTheGinus)
delay: 480
backup-thread-priority: 0
keep-count: 15
local-keep-count: 1
backups-require-players: true
scheduled-backups: false
schedule-timezone: -00:00
backup-format-timezone: -00:00
backup-list:
- path: world
  format: '''Backup-world-''yyyy-M-d--HH-mm''.zip'''
  create: true
- path: plugins
  format: '''Backup-plugins-''yyyy-M-d--HH-mm''.zip'''
  create: true
  blacklist: ['dynmap/**']
dir: backups
destination: backups
googledrive:
  enabled: true
onedrive:
  enabled: false
ftp:
  enabled: false
  hostname: ftp.example.com
  port: 21
  sftp: false
  ftps: false
  username: Username
messages:
  send-in-chat: true
  no-perm: You don't have permission to do this!
  backup-start: Creating backups, the server may lag for a little while...
  backup-complete: Backup complete,
  next-backup: the next backup is in %TIME minutes
  next-schedule-backup: the next backup is at %DATE
  next-schedule-backup-format: h:mm a EEE, MMM d O
  auto-backups-disabled: Automatic backups are disabled
advanced:
  metrics: true
  update-check: true
  suppress-errors: false
  prefix-chat-messages: true
  date-language: en
  ftp-file-seperator: /

**Comment by MaxMaeder** [comment]
@rrunswickjr Thanks, let me know if it works

**Comment by rrunswickjr** [comment]
Got a new error:
[08:49:13 INFO]: [DriveBackupV2] Forcing a backup
[08:49:13 WARN]: Exception in thread "Thread-28" java.lang.Error: Unresolved compilation problems:
[08:49:13 WARN]:        Unhandled exception type Exception
[08:49:13 WARN]:        Unhandled exception type Exception
[08:49:13 WARN]:        Unhandled exception type Exception
[08:49:13 WARN]:        Unhandled exception type Exception
[08:49:13 WARN]:
[08:49:13 WARN]:        at ratismal.drivebackup.UploadThread.run(UploadThread.java:127)
[08:49:13 WARN]:        at java.lang.Thread.run(Unknown Source)

**Comment by MaxMaeder** [comment]
Can you re-download from the link? I think I fixed the issue.

**Comment by rrunswickjr** [comment]
I will report when the backup completes, but it appears to be working at this time.  

What is the best place to ask you an unrelated question about local backup destinations?

**Comment by rrunswickjr** [comment]
Backup just completed, looks good.  Thank you

**Comment by MaxMaeder** [comment]
Will be fixed in 1.3.5.
